utils/batch_key.go

```diff
@@ -0,0 +1,97 @@
+package utils
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+const (
+	// TODO: move to a common constatns file which
+	// can be used by utils and cmd
+	DefaultKeyFolder  = "keys"
+	PasswordFile      = "password.txt"
+	PrivateKeyHexFile = "private_key_hex.txt"
+)
+
+// ReadBatchKeys reads the batch keys from the given folder
+// and returns the list of BatchKey
+// folder: folder where the keys are stored, relative to the current directory
+func ReadBatchKeys(folder string, isECDSA bool) ([]BatchKey, error) {
+	var batchKey []BatchKey
+	absFolder, err := filepath.Abs(folder)
+	if err != nil {
+		return nil, err
+	}
+
+	// read the private key file
+	privateKeyFile, err := os.Open(absFolder + "/" + PrivateKeyHexFile)
```

Comment:
this path won't work on windows. are we fine with that? otherwise use os.path or whatever golang stdlib for paths

Reply:
I guess. we can use a stdlib. there's a filepath.Clean utility I can use.